db/migrations/000012_create_post_likes.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS post_likes
+(
+    id         serial PRIMARY KEY,
+    like_id VARCHAR UNIQUE NOT NULL,
+    user_id    INTEGER REFERENCES users (id),
+    post_id INTEGER REFERENCES posts(id)
+);
```

Comment:
Can we directly add the indices for the `post_likes` (both user_id and post_id)?
We will need that anyway later

Reply:
yeah sure